config/initializers/assets.rb

```diff
@@ -9,3 +9,5 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
 # Rails.application.config.assets.precompile += %w( search.js )
+
+Rails.application.config.assets.precompile += %w( skycons.js )
```

Comment:
Not sure why rails was complaining about that, I thought `require_tree .` would grab it. But removed this anyway after moving to vendor